test/ufc/test_expression.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019 Michal Habera
+#
+# This file is part of FFC (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+
+import numpy as np
+import cffi
+
+import ufl
+import ffc.codegeneration.jit
+
+
+def float_to_type(name):
```

Comment:
Could you add more comments to the test to help someone understand how to use the new functionality?